src/components/Book.css

```diff
@@ -0,0 +1,139 @@
+#root {
+  padding: 0;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  margin: 6rem;
+}
+
+.title-book {
+  color: gray;
+}
+
+.Book {
+  display: flex;
+  gap: 8rem;
+  margin: 0 6rem;
+  align-items: center;
+  margin-top: 4rem;
+  border: 1px solid lightgray;
+  padding: 2rem;
+}
+
+.porcent {
+  margin: 0;
+  font-weight: normal;
+}
+
+.porcent-name {
+  margin: 0;
+}
+
+.div-form {
+  display: flex;
+  flex-direction: column;
+  margin: 6rem 6rem;
+  border-top: 2px solid lightgray;
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+  gap: 2rem;
+}
+
```

Comment:
- [ ] It would be better if we implement the require design as in the screenshot below. Kindly check and compare and make the necessary styling
`developer`
![Screenshot from 2022-03-05 14-52-28](https://user-images.githubusercontent.com/46388113/156888893-a0ce8398-b5d5-4fdf-99f8-234f1653ff5b.png)

`required`
![Screenshot from 2022-03-05 14-52-43](https://user-images.githubusercontent.com/46388113/156888906-b9233e55-20e1-45f9-a38b-6d967df8e328.png)
Kindly consider padding, margin, positioning, etc